Cargo.toml

```diff
@@ -104,6 +104,7 @@ resource-assembler = []
 
 [lints.clippy]
 len_zero = "allow"
+uninlined_format_args = "allow"
```

Comment:
should we put this to a dedicated PR?
with live redirect tests disabling

Reply:
was handled by #502